### ISSUE TITLE
Cannot be configured

### ISSUE THREAD

**Issue by serbasan** [sman591/homebridge-lg-thinq-ac#1]
Hi. It doesn't show any url in settings.
By another method I have obtained both url but have not completed the rest of the fields either. The configuration has not worked at all. Can I help with something?

**Comment by sman591** [comment]
Hey @serbasan do you see any errors or other context in the Homebridge logs?

There's an outstanding issue with the homebridge import that I hope to have a fix up tonight for

**Comment by serbasan** [comment]
Hi- I used Docker homebridge.

```
[5/11/2020, 8:27:13 PM] Error loading platform requested in your config.json at position 4
[5/11/2020, 8:27:14 PM] Error: The requested platform 'LgThinqAirConditioner' was not registered by any plugin. 
at PluginManager.getPluginForPlatform (/usr/local/lib/node_modules/homebridge/src/pluginManager.ts:223:15)
at /usr/local/lib/node_modules/homebridge/src/server.ts:386:37at Array.forEach (<anonymous>)
at Server.loadPlatforms (/usr/local/lib/node_modules/homebridge/src/server.ts:373:27)
at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:152:29)
at cli (/usr/local/lib/node_modules/homebridge/src/cli.ts:78:10)
at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
at Module._compile (internal/modules/cjs/loader.js:1158:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
at Module.load (internal/modules/cjs/loader.js:1002:32)
Starting homebridge in insecure mode...
Starting homebridge in debug mode...
```

```
[5/11/2020, 8:27:17 PM] ====================
[5/11/2020, 8:27:17 PM] ERROR LOADING PLUGIN homebridge-lg-thinq-ac:
[5/11/2020, 8:27:17 PM] Error: Cannot find module 'homebridge'
Require stack:
- /homebridge/node_modules/homebridge-lg-thinq-ac/dist/platformAccessory.js
- /homebridge/node_modules/homebridge-lg-thinq-ac/dist/platform.js
- /homebridge/node_modules/homebridge-lg-thinq-ac/dist/index.js
- /usr/local/lib/node_modules/homebridge/lib/plugin.js
- /usr/local/lib/node_modules/homebridge/lib/pluginManager.js
- /usr/local/lib/node_modules/homebridge/lib/api.js
- /usr/local/lib/node_modules/homebridge/lib/index.js
- /usr/local/lib/node_modules/homebridge/lib/cli.js
- /usr/local/lib/node_modules/homebridge/bin/homebridge
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/homebridge/node_modules/homebridge-lg-thinq-ac/src/platformAccessory.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
[5/11/2020, 8:27:17 PM] ====================
```

**Comment by sman591** [comment]
Ahh yep, this is the `homebridge` import issue.

Is this with the Onzu image? Do you have a `package.json` in your setup? If so, can you try adding `"homebridge": "1.0.4"` to the dependencies array?

Otherwise I'll have a permanent fix for this later tonight

**Comment by sman591** [comment]
RE Spain -- this is also a feature I haven't built yet (I only started on this a few days ago). I hope to look at this soon but it may be a day or two

**Comment by serbasan** [comment]
Yes. It is oznu / homebridge with homebridge 1.0.4 and Homebridge Config UI X. I'll be watching for your updates to test. Thank you for your work.

**Comment by sman591** [comment]
Startup error fixed in 4cfbe66f2d2e3ddbd50a85471398c71f206d78ec

I don't have time to add multi-region support today but I'll update #2 as soon as I do, hopefully in the next day or two

**Comment by sman591** [comment]
@serbasan I just published version 0.3.1 which includes multi-region support. Could you try installing again? I've included details in the README for how to specify the correct country & language code

**Comment by serbasan** [comment]
Hi. The setup has worked. The problem now must be that my AC is not compatible.

[LgThinqAirConditioner] Getting device status d281f580-71.......

[5/13/2020, 12:11:06 PM] [LgThinqAirConditioner] Error during interval update TypeError: Cannot read property 'airState.operation' of undefined
[5/13/2020, 12:11:25 PM] [LgThinqAirConditioner] Triggered SET Active: 1
[5/13/2020, 12:11:25 PM] [LgThinqAirConditioner] Failed to set power state on Error: Request failed with status code 400
